tests/foreman/api/test_registration.py

```diff
@@ -262,3 +262,61 @@ def test_negative_capsule_without_registration_enabled(
         "Proxy lacks one of the following features: 'Registration', 'Templates'"
         in context.value.response.text
     )
+
+
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_host_registration_with_non_admin_user(
+    module_org,
+    module_location,
+    module_activation_key,
+    module_target_sat,
+    rhel_contenthost,
+):
+    """Verify host registration with non admin user
+
+    :id: 02bdda6a-010d-4098-a7e0-e4b5e8416ce3
+
+    :steps:
+        1. Enable the required repositories on the satellite and get a host(RHEL7/RHEL8/RHEL9)
```

Comment:
Just to keep it short
```suggestion
        1. Sync the content repositories, add and publish it in CV
```